OrbitCore/Path.cpp

```diff
@@ -252,7 +243,7 @@ std::string Path::GetProgramFilesPath() {
 #ifdef WIN32
```

Comment:
This one also seems unused - only mentioned in dump.

Reply:
The full list of unused functions:

- GetOrbitAppPdb()
- GetLicenseName()
- GetTmpPath()
- GetProgramFilesPath()
- GetMainDrive()
- GetSourceRoot() - one more occurrence in PathTest.cpp
- IsSourceFile()

Should I remove them all?

Reply:
Yes!